Objects/Objects/Other/Block.cs

```diff
@@ -21,7 +21,7 @@ public BlockDefinition(string name, List<Base> geometry, Point basePoint = null)
   }
 
   public string name { get; set; }
-
+  public string anonymousName { get; set; }
```

Comment:
If you really need the anonymous name property, instead of adding it to the `BlockDefinition` class (which is used by other applications like Rhino and Sketchup that do not have anonymous names), please add it as a dynamic property attached to the block definition instead